eng/test-job.yml

```diff
@@ -98,7 +99,8 @@ jobs:
     condition: and(succeeded(), ${{ parameters.condition }})
 
     # Test job depends on the corresponding build job
-    dependsOn: ${{ format('build_{0}_{1}_{2}', parameters.osIdentifier, parameters.archType, parameters.buildConfig) }}
+    ${{ if ne(parameters.ignoreDependencyOnBuildJobs, true) }}:
```

Comment:
With this change I believe we no longer need this. The entire stage will depend on the product build.

Reply:
Ignore this, it is still required for the ci.